lib/index.js

```diff
@@ -824,25 +824,25 @@ function proxy (frontend, backend, capture){
 
         //forwarding router/dealer pack signature: id, delimiter, msg
         frontend.on('message',function (id,delimiter,msg){
-          backend.send([id,delimiter,msg]);
+          backend.send([].slice.call(arguments));
         });
 
         backend.on('message',function (id,delimiter,msg){
-          frontend.send([id,delimiter,msg]);
+          frontend.send([].slice.call(arguments));
 
           //forwarding message to the capture socket
-          capture.send(msg);
+          capture.send([].slice.call(arguments, 2));
```

Comment:
why the `2` here?

Reply:
to keep original behavior, capture socket doesn't receive identity and delimiter frames

Reply:
right on :)